safe-global/safe-wallet-web
# fix: clear next pages when switching Safe

## What it solves

Resolves stale transaction and messages from previous Safe.

## How this PR fixes it

When switching Safe, every page of transactions/messages after the initial is cleared.

## How to test it

1. Load a subsequent page of transactions on the history/queue or messages and switch Safe.
2. Observe no stale transactions/messages from the previous Safe.

## Checklist
